packages/core-bridge/ts/index.ts

```diff
@@ -458,6 +459,10 @@ export interface TimeSkippingServerConfig {
    * be supported in the future.
    */
   extraArgs?: string[];
+  /**
```

Comment:
I don't think the Time Skipping server supports the `--searchAttribute` option.

Reply:
Didn't realize that - removed.